x-pack/legacy/plugins/reporting/export_types/png/server/execute_job/index.js

```diff
@@ -40,7 +40,7 @@ function executeJobFn(server) {
       }),
       map(omitBlacklistedHeaders),
       map(getConditionalHeaders),
-      mergeMap(addForceNowQuerystring),
+      mergeMap(getFullUrls),
```

Comment:
Nice rename here

Reply:
It was still surprisingly hard to come up with ;)